wolfSSL/src/chapter02.md

```diff
@@ -484,6 +648,22 @@ Enables SHA-384 support.
 
 Enables SHA-512 support.
 
+#### WOLFSSL_AES_128
```

Comment:
These are all on by default. The macro that should be documented are `NO_AES_128`, `NO_AES_192` and `NO_AES_256`.